site/src/helpers/constants.ts

```diff
@@ -0,0 +1,2 @@
+// SearchPage
+export const PAGE_SIZE = 10;
```

Comment:
I would suggest renaming this to something like `NUM_RESULTS_PER_PAGE` , but I'll leave this up to you :)